src/components/Slider/SliderFixedRange.tsx

```diff
@@ -267,7 +266,14 @@ const SliderFixedRange: React.FC<SliderFixedRangeProps> = ({
             onPressPoint={(point: number) => tapToSeek && onPressPoint(point)}
           />
         )}
-        <Tracked style={[trackedStyle, animatedTrackStyle]} />
+        <Track
+          style={StyleSheet.flatten([
+            styles.track,
```

Comment:
The styles.track is StyleSheet.absoluteFillObject so we do not need create this style